webapp/components/notifications/Notification/Notification.vue

```diff
@@ -1,14 +1,8 @@
 <template>
-  <ds-space :class="[{ read: notification.read }, notification]" margin-bottom="x-small">
+  <ds-space :class="{ read: notification.read, notification: true }" margin-bottom="x-small">
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-30T12:36:12Z" title="Friday, August 30th 2019, 2:36:12 pm +02:00">Aug 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,14 +1,8 @@
 <template>
-  <ds-space :class="[{ read: notification.read }, notification]" margin-bottom="x-small">
+  <ds-space :class="{ read: notification.read, notification: true }" margin-bottom="x-small">
```


what is the justification for this change?
@alina-beck suggested in a `PR` review to me that this could be refactored to the way it is now...

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-30T13:54:47Z" title="Friday, August 30th 2019, 3:54:47 pm +02:00">Aug 30, 2019</time>_

---


It was just broken. If you want to generate css classes from a javascript object, you have a mapping `className->booleanValue`. The way it was before stringified the `notification` object and added it to the `class` attribute. I'm sure this is *not* what we want.